rails-homework/app/views/physicians/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+  <span><%= @physician.first_name  %></span>
+  <span><%= @physician.last_name  %></span>
+  <div><%= @physician.phone_number %></div>
+  <div><%= @physician.level %></div>
+  <button><%= link_to "Edit", edit_physician_path(@physician) %></button>
+  <button><%= link_to "Delete", physician_path(@physician), method: :delete, data: { confirm: "Are you sure?" } %></button>
```

Comment:
This code will generate <button> with <a> inside. It is not a good approach. If you want a button then use `button_to` helper. If you need a link (<a>) with `button` class then pass it to parameters of `link_to` function call.